.ci/setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -ex
+
+sudo yum install -y git
```

Comment:
we will need this until [grok filter is released](https://github.com/logstash-plugins/logstash-filter-grok/pull/162) - since they are interdependent on each other